generate.js

```diff
@@ -1,19 +1,40 @@
-const faker = require('faker');
-const mongoose = require('mongoose');
-const db = require('./models/');
-const Amenity = require('./models/Amenity.js');
-const Room = require('./models/Room.js');
+const faker = require("faker");
+const mongoose = require("mongoose");
+const db = require("./models/");
+const Amenity = require("./models/Amenity.js");
+const Room = require("./models/Room.js");
```

Comment:
In this piece of code here we are requiring threee diffeerent objects out db, Amenity, and Room.
but if we look at our code in `models/index.js` we can see that when we require this file it exporting  Amenitys and Rooms for us already.

so this means technically we could change these three lines to match better.

OLD:
``` 
const db = require("./models/");
const Amenity = require("./models/Amenity.js");
const Room = require("./models/Room.js");
```

NEW v1: 
```
const db = require("./models/"); //exporting amenitys and rooms already
const Amenity = db.Amenity;
const Room = db.Room;
```

or we can use it by

```
const db = require("./models/");

db.Rooms.find()...
db.Amernity.find()...
```

This piece of code is technically not wrong so if you like this style better I say use it.